en/docs/using-wso2-identity-server/account-locking-by-failed-login-attempts.md

```diff
@@ -133,9 +112,10 @@ descriptions you need to configure:
 <p>Account unlock timeout = Configured <strong>Account Unlock Time</strong> * ( <strong>Lock Timeout Increment Factor</strong> ^ failed login attempt cycles)</p>
 <p>i.e.,</p>
 <p>10 minutes = 5 * ( 2 ^ 1 )</p>
-!!! tip
-    <p>Tip</p>
-    <p>If you want to configure the <strong>Lock Timeout Increment Factor</strong> property via the file based configuration, the parameter you need to configure is <code>                 account.lock.handler.login.fail.timeout.ratio                </code> found in the <code>                 &lt;IS_HOME&gt;/repository/conf/identity/identity-event.properties                </code> file.</p>
+<div class="admonition tip">
+<p class="admonition-title">Tip</p>
+    <p>If you want to configure the <strong>Lock Timeout Increment Factor</strong> property via the file based configuration, the parameter you need to configure is <code>                 auto_unlock_time_increment_ratio                </code> found within     
+    <code>  identity_mgt.account_locking </code>   in the <code>                 &lt;IS_HOME&gt;/repository/conf/deployment.toml                </code> file.</p></div>
```

Comment:
shouldn't we ideally say to add this parameter into the toml file? shall we give it as a code block like we usually give toml configs? WDYT?